src/compute-repository.js

```diff
@@ -123,7 +123,9 @@ module.exports = async function (specs, options) {
    * default branch of the GitHub repository associated with the specification.
    */
   async function determineSourcePath(spec, repo) {
-    // Retrieve all paths of the GitHub repository
+    // Retrieve all paths of the GitHub repository, excluding symlinks
+    // (note that, while we may identify symlinks, there is no easy way to
+    // get the target of the symlink from the GitHub API)
```

Comment:
how about https://developer.github.com/v3/repos/contents/#response-if-content-is-a-symlink?

Reply:
hmm... I had misread; it only shows the target file is the file is NOT in the repo

Reply:
the Git Tree API gives it though: https://api.github.com/repos/w3c/webrtc-pc/git/blobs/master links to https://api.github.com/repos/w3c/webrtc-pc/git/blobs/8ddfdec7c40e6daee22a0b79df05546645e20b2a for the index.html file with a content that decodes to "webrtc.html"

Reply:
I would argue that qualifies as "no easy way" ;) I'll include that logic.